dx_workflows/vgp_falcon_and_unzip_assembly_workflow/Readme.md

```diff
@@ -2,7 +2,12 @@
 ## Update note
 
 **2018-Jan-23patch1 update**
-- new TANmask and REPmask is very buggy, so I roll back TAN and REPmask from 2.0.0 to 1.9.3. That version has difference name for standard, tanmask, and repmask, but us old Falcon code.
+- start using TANmask2.0.0 and REPmask2.0.1 after testing on 3 large genomes
+- fix discrepancy of unzip_stage_0 app and github. The old version miss logic to handle missing ref/read. New version is unzip_stage_0 track_read 1.1.0
```

Comment:
Can you clarify what the issue was that was fixed by this?

Reply:
At the end of unzip_stage_0, there will be pairs of read and ref files created. In some corner case, a member of a pair was not created. The reason why they are not created is always clear, but the hypothesis from Pacbio team is there is a conflict in assign rew ->pread-> contig that doesn't have a tree structure. Therefore, we put logic to get rid of an unpair couple and print a message for inspection around June last year, but the code doesn't make it to the app for an unknown reason.

Reply:
Ok, thanks